tools/cdk-release/lib/updaters/types/json.ts

```diff
@@ -0,0 +1,27 @@
+import { UpdaterModule } from '../../types';
+
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+const detectIndent = require('detect-indent');
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+const detectNewline = require('detect-newline');
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+const stringifyPackage = require('stringify-package');
+
+const jsonUpdaterModule: UpdaterModule = {
```

Comment:
Not just ANY json... specifically `packageJson`, no?

Reply:
Actually, any JSON with a `"version"` key - it doesn't have to be `package.json`!

This is especially important for our use case, since we store our versions in [`version.v1.json`](https://github.com/aws/aws-cdk/blob/c3096ea6c36ec5c89a7f439b254c26107cc8cb0a/version.v1.json#L1-L3) and [`version.v2.json`](https://github.com/aws/aws-cdk/blob/a0dc6fa6a89090b9744b1806fbfadb6b5fb57302/version.v2.json#L1-L3) files.

Reply:
Yeah okay. I guess I expected a class like:

```ts
class JsonUpdater implements UpdaterModule {
    constructor(private readonly versionFieldName: string) {
    }

   // ...
} 
```

But also: 💅 don't feel strongly about this

Reply:
Changed.